### DISCUSSION TITLE
개발팀 데일리 아젠다

### DISCUSSION THREAD

**Discussion by ipdae** [planetarium/NineChronicles#6571] [General]
개발팀 데일리에서 논의해야할 아젠다 리스트

**Comment by sonohoshi** [comment]
https://github.com/planetarium/NineChronicles/issues/6238 이거 시연 후 해야하는 작업별로 일정 산정해서 다시 estimate랑 end date 수정해뒀습니다.

**Comment by eugene-doobu** [comment]
토르 클라이언트 작업 l10n / LiveAsset으로 데이터 관리 / 체인 상관없이 이벤트 열리면 이벤트 UI활성화 작업 어제 진행했어서 완료일 어제로 수정했습니다 
https://github.com/planetarium/NineChronicles/issues/6148

**Comment by ipdae** [comment]
https://www.notion.so/planetarium/12ded889905f809fa9b2c2e9489d7593 공유

**Comment by U-lis** [comment]
[여기](https://github.com/planetarium/NineChronicles.SeasonPass/pull/188/commits/9b531c3d512c5801f1e5e249115b4781689a946e) 에 중간에 보면 avatar 별로 state 를 들고 와야 하는 로직이 있는데, 이게 headless 부하를 많이 줄 것 같아서 부하를 좀 줄일 수 있는 방법이 있을 지 논의해보고 싶습니다.
(context 및 기능 설명 필요)

---

- GQL 로 state 를 한번에 가져올 수 있게 하는 endpoint 를 만들거나
- RPC 로 불러서 가져올 수 있게 하거나 (이게 GQL 보다 빠를것)
- headless renderer 가 내부적으로 캐싱해서 가지고 있도록 하거나

**Comment by U-lis** [comment]
[World clear pass](https://github.com/planetarium/NineChronicles.SeasonPass/issues/182) 작업 일정이 그제로 되어 있었는데 adventure boss pass 및 API 수정 등 손봐야 할 게 많아서 일정이 지연되고 있습니다. 오늘 저녁까지 작업해서 월요일 오전에는 리뷰 보실 수 있도록 하겠습니다.

**Comment by ipdae** [comment]
https://github.com/planetarium/NineChronicles/issues/3932 논의

---
- 클라이언트의 1차적인 작업을 했던 기록이 남아있음
- 아우라관련 DP에서의 처리가 필요함 -> @area363 에게 문의 필요
- 룬은 구현이 안되있음

**Comment by ipdae** [comment]
#3932 팔로업

---
- 테이블 추가 후 @area363 께 요청
- 아우라, 그리모어는 큰 문제 없어보임
- 테이블 추가는 언제쯤? -> rc기간내(이번주) 처리

**Comment by ipdae** [comment]
RC 관련 체크
- [x] 토르 제네시스
- [x] 릴리즈 브랜치
---
- 립플래닛 의존성 변경으로 헤드리스 작업중
- DP 액션 기록 남겨야함

**Comment by ipdae** [comment]
https://github.com/planetarium/NineChronicles.IAP/pull/333 시즌 패스 구조

---
- 스토어와 서비스의 중복 등록 휴먼 에러에 대한 고민
- 검증의 경우 지급된 tx를 확인하는 식으로 밸리데이션 필요
- 현재도 상품 등록시에는 여전히 개발자(@U-lis)의 손을 거쳐야함
- 외부서비스를 통해 등록된 리스트 확인은 가능함
- 모델변경 대응이 필요한 상황

**Comment by sonohoshi** [comment]
https://planetariumhq.slack.com/archives/C07R2JW2KND/p1731472410385199?thread_ts=1731390243.064759&cid=C07R2JW2KND 이런 문맥으로 #6307 일정이 오늘 오후까지 추가 작업을 해야할 것 같습니다. end date는 수정해뒀습니다

**Comment by ipdae** [comment]
- #6310 
- #6290 
이슈 팔로업
---
- 이슈 어사인 변경
- 파티클 크기 변경 문의?

**Comment by ipdae** [comment]
- 전투공식 밸런스 관련 https://planetariumhq.slack.com/archives/CBU2JFAF3/p1731573487292059

**Comment by ipdae** [comment]
- IAP 상품정보 클라이언트 처리 현황
- IAP 관련 규칙정하기?

**Comment by ipdae** [comment]
- 시즌패스 팔로업
- 월드보스 미래 대비
- 토르 모바일 플렛지

---
- https://github.com/planetarium/NineChronicles.SeasonPass/issues/175 이번주안에 메인넷 대상으로 노드 부하 테스트 진행
- 월드보스 DP 의존성 제거 진행 예정
- 기존 오딘 또는 헤임달에 가입된 계정이 토르에 신규 생성시 오류가 발생중 @eugene-doobu 님이 보시는거 서포트 @sonohoshi @tyrosine1153

**Comment by U-lis** [comment]
internal checklist 만들기 문서가 다른 대응들에 밀려서 자꾸 못하고 있네요
이거 완료일을 아예 한 26일정도로 밀어놓고 작성하도록 하겠습니다

**Comment by ipdae** [comment]
- QA보드 정리 확인
- due, end date 확인

**Comment by sonohoshi** [comment]
- https://github.com/planetarium/NineChronicles/issues/6402 이거 좀 밀려서 0.25일에서 0.5일로 수정했습니다. 어제 오후에 하려고했는데 미팅 이것저것 들어가느라 밀렸어요

**Comment by ipdae** [comment]
- https://github.com/planetarium/lib9c/pull/3022 어사인 및 추가 작업
- [x]  DRR계산시 최대값을 81%로 설정
- [x] BuffLimitSheet에서 절대값을 설정가능하도록 작업

**Comment by U-lis** [comment]
RC 및 preview 빌드 뽑는 일정 체크 필요
우리 지금 QA 진행중인 버전이 있던가요? 이전버전 끝났고 아직 수정된거 전달을 안한 것 같은데

**Comment by ipdae** [comment]
- 사인오프 프로세스 공유 https://www.notion.so/planetarium/2024-11-20-sign-off-process-draft-144ed889905f80378854d2bebe10157e
- QA 상황 확인
- @U-lis IAP 데이터 규칙 현황 확인
- @eugene-doobu 스킬밸런스 툴 현황 확인

---
사인오프
- pr 서머리를 사람손말고 자동화할 방법은 없을지 
- 적용 예정 버전은 260(다음 릴리즈)부터 예정
스킬밸런스 툴 
- 아바타 상태 만들기 discussion https://github.com/planetarium/NineChronicles/discussions/6451
- 클라이언트 툴 주단위로 예상. 에디터 UI, 인자 커스터마이징이 오래걸릴 것으로 예상
- @NineSyu 의 요구사항 초안 공유 필요
- 복잡도를 낮추고 MVP는 최대한 간단한 스펙(내스탯, 상대스탯, 사용스킬)으로 적용하면 좋을것 같음

**Comment by ipdae** [comment]
토르 메인넷 준비 상황 확인
---
- @jonny-jeahyunchoi 모바일 250.0.8을 메인넷 바라보도록 수정
- @ipdae pc 포탈 준비되면 테스트 후 배포

**Comment by jonny-jeahyunchoi** [comment]
250.0.9 빌드 준비 확인

**Comment by ipdae** [comment]
- rc 백머지, 태깅
- 토르 초기화
- 업무방식 변경 관련
- dev 룰 추가 관련 https://www.notion.so/planetarium/lib9c-14bed889905f80bd972ff15064265b42

**Comment by eugene-doobu** [comment]
260 rc 일정

**Comment by ipdae** [comment]
- 프로토타이핑, 9c 인원분배 관련
- 아레나
- 260 준비
- 런쳐 인수인계

---
- 클라이언트 2명이 차출되면 아무래도 힘들지 않을까
- @ipdae 런쳐인수인계
- @eugene-doobu 합성 기본 UI작업도 필요해서 분업은 힘든 상황
- 전투공식제한 -> 미정
- @sonohoshi 몬스터콜렉션
- DP 데이터 기록 추가
- IAP, 시즌패스 논스가 꼬였을때 대응
- 신규 상품등록
- 다음주 화요일 주간미팅에서 프로토 타이핑 관련 논의 및 멤버가 정해지면 좋을듯

**Comment by ipdae** [comment]
- 프로토타이핑
    - RNG 류를 만들 예정
    - 밈코인 같은게 꼭 붙을 것 같다.
    - BE1 FE1 구성. 아마 V8 에 연동되게 할 듯.
    - Pet simulator (로블록스)
    - 핵심은 뽑기 전투 거래 랭킹(경쟁) global noti 채팅?
- 정찰보상 장애 부검
    - 어제부터 잘 작동하지 않고 있음. 알림이 오지 않아서 아직 복구중.
- 사인오프
    - commit 의 형식화 - sign off changelog 는 기계적으로 할 수 있을듯.
    - backmerge 할 때 전체 변경점을 template 으로 만들어서 confirm 받고 들어간다.
    - 자동화 툴 없으면 수동으로.

**Comment by ipdae** [comment]
- 250 마일스톤 아카이빙 처리 공유
- 주간미팅 시간 변경
- 현님 우선순위 변경
- 260 준비

---
- dev 주간미팅 화요일 진행
- 통합 마일리지 클라이언트는 처리 문제 없음
- 이전 테이블을 물리적으로 분리시키면 좋겠음
- 260 배포일 20일

**Comment by ipdae** [comment]
- https://www.notion.so/planetarium/2024-12-Launcher-v2-152ed889905f80a2b0b9d6743556d0ee#152ed889905f80dfa1cdf0ddf8686512 런쳐 문서 공유
- https://www.notion.so/planetarium/9c-dev-11-153ed889905f800b9b30dca8a016482e 11월 회고
- IAP 최적화
- 신작 BE 요구사항

---
- IAP 이슈화 완료
- 신작 BE 관련 정리 5시까지 완료

**Comment by U-lis** [comment]
Release branch + 내부 200260 테스트 시작 일정 체크

- 머지 필요한 PR 들 있는지 체크
    - [x] lib9c
    - [x] 9c
- headless, DP 체크
    - [ ] headless : lib9c 만 따라가면 됨
    - [ ] DP : 합성 action 기록 (PR review 대기중)

**Comment by ipdae** [comment]
- state proxy 관련 변경 필요
- https://github.com/planetarium/NineChronicles/blob/development/nekoyume/Assets/_Scripts/Blockchain/RPCAgent.cs#L645
- https://github.com/planetarium/NineChronicles.Headless/blob/development/NineChronicles.Headless/BlockChainService.cs#L254

**Comment by eugene-doobu** [comment]
합성 확률 관련해서 문제가 있었다. 랜덤 테스트 관련해서
- 몬테카를로 시뮬레이션처럼 무작위로 테스트 많이돌려서 결과를 뽑아보는 그런게 있어야 되나 싶은데
- 합성 20000뽑 테스트중인데 겁.나. 오래걸립니다. 매 CI나 이런거마다 돌리긴 어렵고 생각날 때 수동으로 돌릴 수 있게 해야할 거 같은데 이게 작업 이후 인지가 될까?

**Reply by ipdae** [reply]
- 1번의 검증은 의미 있다 생각함. 상시로 확인해야하는지는 모르겠음.
- CI에선 안돌도록 Skip을 걸수있긴함(로컬에서만 필요할 경우 직접 돌리도록)
- 정말 중요한 테스트라면 시간보다는 특정 브랜치(릴리즈라던지)를 대상으로 돌리는게 어떨지

**Comment by ipdae** [comment]
- 유저 문의 팔로업 https://planetariumhq.slack.com/archives/CBU2JFAF3/p1733907315945299
- 12월 회식

**Reply by ipdae** [reply]
- 다른건 모두 성공하는데 tx 스테이징만 실패하고 있었음
- 알고보니 헤드리스 입장에서 봇으로 판단하고 IP밴이 됐던 상황
- 헤드리스, 클라이언트 통신에서 밴이 된 경우 인지를 시켜줘야함

**Comment by ipdae** [comment]
- 릴리즈 노트 준비 (사인오프)
- https://github.com/planetarium/lib9c/pull/3058 어사인

**Comment by Atralupus** [comment]
- Mimir를 다 같이 볼 수 있는 서비스가 되어야합니다. 30분정도 쓰면 해결 가능한 간단한 이슈 하나씩 해결하면서 프로젝트 구조 보는 정도로 끝내면 좋을 것 같네요
  - 좀 더 나아가서 지금 dp로 받는 게임 내 랭킹 서비스들을 mimir로 구현하는걸 해봐도 좋을 것 같네요
- 온콜 쉬프트 설정을 해야합니다. 지금은 dx 팀으로 되어있습니다

**Reply by ipdae** [reply]
- 미미르의 인수인계가 필요함. DP비슷하게 게임에서 사용처를 늘린다던지 확장가능하도록 게임팀이 모두 볼 수 있는 상황이 됐으면함
- 인터널에선 테스트가 필요할때빼곤 미미르를 내려놓음(헤드리스가 영향받음)
- 배포방법에 대한 문서화 등이 되있는지? 리드미에 있음. 테라폼즈 사용. https://github.com/planetarium/mimir argocd, 9c-infra같은 툴 준비라던지 필요환경 구축을 다같이 보고 시작하는게 좋을듯.
- @Atralupus 온콜 문서화 보강 필요 https://www.notion.so/planetarium/DX-5bed6db803704605a5b8b1da946c135b
- @ipdae 페이저듀티 스케쥴 게임팀쪽으로 이동시키기

**Comment by ipdae** [comment]
https://planetariumhq.slack.com/archives/CBU2JFAF3/p1734534528070459 아레나 장애상황 관련 대응

**Comment by ipdae** [comment]
- 프리뷰넷 정리
- 개발팀 잘한일

**Reply by ipdae** [reply]
- 번역어, 어드레서블, 메모리 최적화 등 리팩토링
- 9c K 대응
- 최소 QA기간(1주일) 확보
- 토르 런칭

**Reply by ipdae** [reply]
- IAP 최적화(신규 패키지 관리 리소스 최적화)

**Comment by eugene-doobu** [comment]
릴리즈 노트 개선안?

**Reply by ipdae** [reply]
- 클라이언트 릴리즈 준비할때 수동으로 하고 있는게 많음.
- @sonohoshi 님이 1차적인 툴을 만들었으나 현재 작동안함.
- 자동화가 필요함. 다음주 주간미팅이 없으니 규칙에 관련된 논의를 슬랙에서 시작해보면 좋을듯

**Comment by ipdae** [comment]
#6649 이슈 분배

**Comment by eugene-doobu** [comment]
Obsolete된 중복 구현 제거
- 다른건 모르겠고 AddItem같은거부터..

**Reply by ipdae** [reply]
- 대부분 lib9c
- PAEV해서 지워도 되는애들 지우자

**Reply by ipdae** [reply]
https://github.com/planetarium/lib9c/issues/3105

**Comment by ipdae** [comment]
![image](https://github.com/user-attachments/assets/a97f8c51-6d3a-4154-8b90-3ed758844dda)
#6658 어사인 변경

**Reply by ipdae** [reply]
목요일에 논의

**Reply by eugene-doobu** [reply]
- EventRewardPopup에서 액션 렌더받았는지 확인할 방법을 만들어야함(버튼 애니메이션 상태등의 이유로 그냥 ResponseClaimPatrolReward 에서 메소드 하나 호출하기에는 위험해보입니다) -> EventRewardPopup이 액션 하나 보내면 응답 올때까지 버튼클릭 다막고 대기하는 형태면 1번은 뭐 크게 고려 안해도 될 거 같긴 한데.. 이것도 확인 해봐야 하긴 함
- 데이터 초기화 과정 개선필요? UI 생성시점에서 필요한 데이터들이 초기화가 안되있어서 예외가 터지는 상황이라 이건 대강 땜질해두긴 했는데 제대로 확인 필요할 것 같긴 합니다.
- 기타 아이템맵핑이나 patrol time들이 연결이 안되있는상황이긴한데, 이걸 기존에 있던PatrolRewardPopup하고 같이 신경써야할지..

 모델만 수정하면 다 적용이 될지 정확히 모르겠습니다.
- 만약 3번이 아니라면 PatrolRewardPopup와 EventRewardPopup 사이의 교통 정리가 필요할 것 같습니다.

**Reply by ipdae** [reply]
- 1주년 팝업을 계속 사용할건지
- 패트롤 리워드 팝업을 유지할건지
---
주요 변경 내역
- 서비스에 요청을 보내고 폴링으로 기다리던 부분을 직접 액션을 보내고 렌더러에서 해제
- 서비스에 받은 정보와 시간베이스로 표시하던 부분을 체인에서 상태로 확인하고 블록인터벌 사용
- 사용 액션 변경으로 핸들링

**Comment by ipdae** [comment]
#6661 타임라인 정리

**Comment by jonny-jeahyunchoi** [comment]
https://www.notion.so/planetarium/Cursor-16fed889905f808ebf98de18b37137d4
본격적으로 커서를 좀 활용해보려고하는데
9c 래포에 커서관련 ide 패키지 설치해도될까요? 로컬에만 설치해서 사용하다보니 브랜치변경할때마다 스테이징하거나 재설치해야해서 아예 래포로 포함시키면 어떨까합니다
![image](https://github.com/user-attachments/assets/6c354822-442d-485b-a8ed-b83331328118)

**Comment by ipdae** [comment]
- https://www.notion.so/planetarium/9c-dev-2024-170ed889905f802caef2d184c5e6f69c 회고 준비
- https://www.notion.so/planetarium/2025-01-06-9C-170ed889905f8058a107e5c3db02cfb1?pvs=4#170ed889905f8049baaefa71508e913a 모다톤 투표 공유

**Comment by eugene-doobu** [comment]
ActionRenderHandler 코드 간단화?
- 현재 ActionRenderHandler의 라인수가 4600줄을 넘어가고 액션 등록하는 메서드도 점점 길어지고 있는데, 액션별로 렌더과정을 별도 클래스로 파서 하는게 좋지 않을까 생각이 듭니다.
- 액션 등록할때 Validate과정, 에러처리 등을 유동적으로 처리하게 할 수 있으면 될거 같습니다
- Unrender요건 이제 안쓰는거같은데 관련 주석정도는 청소해도 괜찮을까요

**Reply by eugene-doobu** [reply]
뭔가 처리방법이 다른 액션이 있음
```c#
        private void Stake()
        {
            _actionRenderer.EveryRender<Stake>()
                .ObserveOn(Scheduler.ThreadPool)
                .Where(ValidateEvaluationForCurrentAgent)
                .ObserveOnMainThread()
                .Subscribe(ResponseStake)
                .AddTo(_disposables);
        }

        private void ClaimStakeReward()
        {
            _actionRenderer.ActionRenderSubject
                .Where(eval =>
                    ValidateEvaluationForCurrentAgent(eval) &&
                    eval.Action is IClaimStakeReward)
                .ObserveOnMainThread()
                .Subscribe(ResponseClaimStakeReward)
                .AddTo(_disposables);
        }
```

**Reply by U-lis** [reply]
DP 에서도 action 이 늘면서 renderer 가 계속 비대해지는 문제가 있어가지고 이걸 partial class 로 쪼개는 작업을 하고 있습니다.

**Reply by ipdae** [reply]
- 액션마다 구현해야할 성공, 실패등의 케이스건에 대해서 인터페이스로 정리

**Reply by ipdae** [reply]
- Reorg, Unrender등 POW시절의 잔재들은 지우자

**Comment by U-lis** [comment]
(공유) 270 릴리즈 일정과 임시공휴일 - [링크](https://planetariumhq.slack.com/archives/C01D7N32V55/p1736243971889629?thread_ts=1736228360.566209&cid=C01D7N32V55)

**Comment by ipdae** [comment]
페이저듀티 알람 케이스 정하기

**Reply by U-lis** [reply]
https://planetariumhq.slack.com/archives/C06LXJYDA1F/p1736298309579839

**Reply by ipdae** [reply]
시즌패스
- 블록이 밀리는 경우(체인 장애)
- graphql 응답 속도 문제(노드 부하)

**Comment by jaeho0103** [comment]
각 기능별 1차 완료시 내부체크를 위한 커뮤니케이션 프로세스 정리

**Reply by ipdae** [reply]
- 코드 리뷰완료 후(또는 스킵됐을땐 즉시) 1차적인 코드 변경이 없는 상태가 됐을때 QA진행
- 개발팀 데일리 완료후 QA로 이동될 이슈를 넛징

**Comment by eugene-doobu** [comment]
@github-actions
Build-misc / build_lib9c (pull_request) Failing after 1m

실패하는거 점검

**Reply by ipdae** [reply]
lib9c가 더이상 내부에서 서브모듈로 libplanet을 쓰지 않고 있음. 해당 액션 삭제필요

**Comment by ipdae** [comment]
- 개인 작업 로그 공유 및 후기

**Reply by ipdae** [reply]
1주일 더해보기로

**Comment by sonohoshi** [comment]
- 신작 개발은 오픈소스로 진행할지? 아니면 소스를 공개하지 않을지?

**Comment by ipdae** [comment]
- 월드보스 서비스 셧다운 https://planetariumhq.slack.com/archives/C04DPQHUD35/p1736856916545219?thread_ts=1736823506.099139&cid=C04DPQHUD35

**Reply by ipdae** [reply]
- 기존 등급보상 UI를 가능한 재활용해서 기여율별로 정해서 줄 수 있다면 비용을 더 줄일 수 있지 않을지
- UI, 체인상 큰 문제는 없어보임

**Comment by U-lis** [comment]
DCC 에서 9c address 와 crypto address 를 연결하는 방법 질문
portal 계정도 연결할 수 있나?

**Reply by ipdae** [reply]
- DCC 잼님께 문의
 - 포탈 유저 정보의 API가 필요함. 길님께 문의

**Comment by eugene-doobu** [comment]
신작 삽뜨기 관련

**Reply by ipdae** [reply]
1/24 킥오프

**Comment by ipdae** [comment]
270 RC 빌드 시간 확정

**Reply by ipdae** [reply]
- 신규 코스튬 반영
- #6517

**Reply by ipdae** [reply]
3시 진행

**Comment by ipdae** [comment]
테이블패쳐 관련 문의

**Reply by ipdae** [reply]
- 배포는 깃헙페이지에 되있어서 따로 비용청구는 없음
- 기능 고도화는 힘든 상태. 고도화를 할꺼면 다른쪽에 만드는게 나음
- 에디터가 필요하면 오히려 외부에서 서명하는게 힘들다.

**Comment by ipdae** [comment]
- 비용절감 관련 태스크 공유
- [ ] rds db 통합
  - [x] 메인넷 마켓
  - [ ] 인터널 서비스들
- [ ] 미미르 gcp로 이주
- [ ] 노드 부하 줄이기
- [ ] lb 비용 감소를 위해 eks 외부운영중인 서비스 eks로 이주

**Reply by ipdae** [reply]
미미르는 state proxy처럼  state만 짤라서 가져가고 팁만 받아오면 노드 부하를 줄일수 있고, 상태 업데이트는 어렵지 않을듯. latest_state 만 가지고 있을수 있는지 확인 필요.

**Reply by ipdae** [reply]
이주에는 dump후 중단 이전하면 될것으로 보임

**Reply by ipdae** [reply]
- mongo db는 하나만 쓰고 플래닛별로 분산관리중.
- 네트워크 추가시에는 웹서비스만 추가로 띄우면 됨.

**Comment by ipdae** [comment]
- gcp(nine-chronicles-dev) 프로젝트 생성
- https://www.notion.so/planetarium/184ed889905f800a9066ce2eba548947 IAP 상품등록 조사 공유
- 배포 준비 관련
- 아레나 내부 테스트
- 신작 킥오프
- 미미르 테라폼

**Reply by ipdae** [reply]
- IAP 영수증관련 조사도 자동화 필요

**Reply by ipdae** [reply]
- 아레나 내부 테스트는 다음주 금요일 진행 및 세팅

**Comment by U-lis** [comment]
headless 변경 방법 공유 + release note template 의 k8s 배포 체크리스트 변경
- 이제 9c-infra 에서 PAEV 를 먼저 적용하고 argocd sync 를 해야 합니다.
- lib9c tag 를 체크해서 해당 블록 range 에서 사용할 lib9c 버전을 설정한다.
- PAEV 를 해야 하는 블록은 적당한 미래 블록으로 정한다. 
    - 1블록당 8초로 계산. PAEV + headless sync + block check 정도를 할 수 있는 시간 여유가 필요
- block height 를 체크해야 하기 때문에 한번에 한 planet 씩 진행해야 한다.
- patch table 등이 필요한 경우 ↑ 에서 정한 블록이 지나가고 해야 한다.

**Reply by ipdae** [reply]
- [ ] @Atralupus  280 배포시 주의 필요

**Comment by ipdae** [comment]
1월 회고 일정 정하기

**Comment by ipdae** [comment]
https://www.notion.so/planetarium/25-1Q-192ed889905f80138c39de9c21518728

**Comment by ipdae** [comment]
v280 rc 준비

**Comment by ipdae** [comment]
- 비용절감 마이그레이션 관련
- 시즌패스 실패 알람 https://planetariumhq.slack.com/archives/C06LXJYDA1F/p1739211109308789

**Comment by ipdae** [comment]
v200280 릴리즈 준비

**Reply by ipdae** [reply]
- @Atralupus 아레나 서비스 세팅
- @ipdae lib9c, headless 등 백머지
- @jonny-jeahyunchoi 9c 백머지

**Comment by ipdae** [comment]
로드맵 보드 점검

**Comment by ipdae** [comment]
- 배너를 네트워크(메인, 인터널)따라 띄울수있을지?
- #6898

**Reply by ipdae** [reply]
LiveAssetEndPoint에서 대상 엔드포인트 변경가능. 에디터 테스트에선 이걸로 확인 가능할듯

**Reply by ipdae** [reply]
- 클라이언트에서 메인넷인지 아닌지는 구분이 가능한 상황
- https://github.com/planetarium/NineChronicles/blob/41c76fb20d138a1c9c1f7dbd76ccc9b6178e098e/nekoyume/Assets/_Scripts/Game/LiveAsset/LiveAssetManager.cs#L213 를 참고

**Reply by ipdae** [reply]
EventData(json)에 메인넷 플래그같은걸 사용하면 구분 가능할듯

**Comment by Geonpyo999** [comment]
v200290 관련 문의사항
1. 4월 시즌패스 인터널 반영 시점 문의드립니다.
2. 펫팝 연동 관련하여 체크 필요 항목이 있을지 문의드립니다.
3. 랭킹 / 커스텀 제작 횟수는 v200290에서는 동작하지 않는 상태로 가는 부분인지 문의드립니다.

![image](https://github.com/user-attachments/assets/5cb0381e-b70b-43d0-aea0-23bf6b8eabff)
![image](https://github.com/user-attachments/assets/b2b61924-5aff-40fd-b44e-681e3d5e5cf0)
![image](https://github.com/user-attachments/assets/f4bd09ef-2087-4c05-b983-cd9720c0f8ae)

**Reply by ipdae** [reply]
https://github.com/planetarium/NineChronicles/issues/6906

**Comment by eugene-doobu** [comment]
신작 관련 논의 시기?

**Comment by ipdae** [comment]
버전 배포이후 제보 이슈 팔로업
- 월드보스
- 시트 다운로드

**Reply by ipdae** [reply]
에디터에서 sheetBucketUrl이 설정되있을때 시트를 못받음

**Reply by ipdae** [reply]
https://github.com/planetarium/NineChronicles/issues/6947

**Reply by ipdae** [reply]
시트 다운로드는 https 접속 문제 발견

**Comment by ipdae** [comment]
수요일 POC, 금요일 워크샵 관련

**Comment by ipdae** [comment]
https://www.notion.so/planetarium/9-1b9ed889905f80bd95b1c54704ce21dc

**Comment by U-lis** [comment]
Project setting 을 어떻게 하면 좋을까요?
일단 github project 를 통해 관리를 하는게 하던 방식이니 좋을 것 같기는 한데, 지금 보고 있는 project 가 Nine Chronicles 라서 여기에 petpop 이나 차기 신작을 달기가 뭔가 좀 그렇습니다.
project 를 다 따로 파자니 너무 많아져서 한눈에 안 보일 것 같은데, 좋은 방법이 있을까요?

**Reply by ipdae** [reply]
- 각 프로덕트별 프로젝트 생성 @U-lis 
- 프로덕트별 이슈를 통합으로 볼수있는 개발팀용 보드 만들기 @ipdae

**Comment by U-lis** [comment]
차기작을 위한 petpop backend 분리에 관해
이걸 슬슬 하면 되나 싶었는데 지난주 A8 이 들어오면서 이걸 어떻게 해야 하나 좀 고민이 생겼습니다.
아마 위에서는 계속 A8 을 가능한 활용하길 바랄 것 같은데 A8 이 자체적으로 BE 로직도 포함하는지 어쩌는지 잘 모르겠어서요.
이거 분리해야 한다고 결정나면 petpop 오픈하기 전에 얼른 BE 전용 repo 하나 만들어서 naming rule 정하고 petpop 부터 이전해놓는게 좋을 것 같습니다.

**Reply by ipdae** [reply]
서버와 클라이언트는 우선 저장소를 분리하는것으로

**Comment by ipdae** [comment]
백엔드 배포 환경 체크(aws에 배포되있는지 등)

**Reply by ipdae** [reply]
- 지금은 aws pl-dev에 ec2띄워서 쓰는중(테스트용)
- fe 빌드도 s3, cf(cloudfront) 사용중
- db, redis
-> 이번주 안으로 세팅

**Comment by Atralupus** [comment]
CP 상태가 쌓일 시간이 필요한데 아레나 끝나고 노드만 배포 진행할 수 있을까요

**Comment by Atralupus** [comment]
https://planetariumhq.slack.com/archives/CC7BWJYTZ/p1744048627098629?thread_ts=1744000852.262979&cid=CC7BWJYTZ
커뮤니티 노드 인프라

**Comment by ipdae** [comment]
펫팝 로그 확인

**Comment by U-lis** [comment]
- PetPop quest claim 의 중복 요청이 처리되는 건에 대해서 도움을 주시면 좋겠습니다. (https://github.com/planetarium/petpop/issues/485)
- 서버/DB 부하 테스트는 hoxy 어떻게 되었을까요? :pray:

**Reply by ipdae** [reply]
DB부하 테스트 예정
